tests/conftest.py

```diff
@@ -17,10 +17,18 @@
 from devito.ir.iet import Iteration
 from devito.tools import as_tuple
 
+try:
+    from mpi4py import MPI  # noqa
+    no_mpi = False
+except ImportError:
+    no_mpi = True
+    pass
 
 skipif_yask = pytest.mark.skipif(configuration['backend'] == 'yask',
                                  reason="YASK testing is currently restricted")
 
+skipif_mpi = pytest.mark.skipif(no_mpi, reason="mpi not installed")
```

Comment:
instead of adding this, I'd prefer to leverage the more general mechanism in #637 : https://github.com/opesci/devito/pull/637/files#diff-484462fced51d1a06b1d93b4a44dd535R21

we could rename if ``skipif`` and allow it to digest ``yask``, ``ops``, ``nompi`` -- with "nompi", it does the check that if there's no MPI available, it skips the test

Reply:
Sure but not the point of this PR

Reply:
at least let's call it ``skipif_nompi``...